CHANGELOG.md

```diff
@@ -120,6 +120,11 @@ configuration is as follows:
   Reduce the default number of CPU cores that are used for verifying incoming ATXs to half of the available cores.
 
 * [#5462](https://github.com/spacemeshos/go-spacemesh/pull/5462) Add separate metric for failed p2p server requests
+* [#5517](https://github.com/spacemeshos/go-spacemesh/pull/5517)
```

Comment:
That change will be in v1.3.8 not v1.3.6 😉

Reply:
Fixed, thanks!